ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -181,6 +186,22 @@ export class PrepareMigrationComponent implements OnInit {
     )
   }
 
+  endMigration() {
+    let payload: ISourceAndTargetDetails = {
+      SpannerDatabaseName : this.resourcesGenerated.DatabaseName,
+      SpannerDatabaseUrl: this.resourcesGenerated.DatabaseUrl,
+      SourceDatabaseType: this.sourceDatabaseType,
+      SourceDatabaseName: this.sourceDatabaseName
+    }
+    let dialogRef = this.dialog.open(EndMigrationComponent, {
+      width: '30vw',
```

Comment:
Use standard CSS for all dialog boxes

Reply:
The css for dialog boxes sizing is a little different and cannot be defined using css files. https://stackoverflow.com/questions/40595276/angular2-material-dialog-css-dialog-size. Tried using css by specifying in css file but it doesn't work.